apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -309,6 +311,37 @@ const WorkshopForm = React.createClass({
     }
   },
 
+  shouldRenderRegionalPartnerSelect() {
+    return window.dashboard.workshop.available_regional_partners.length;
+  },
+
+  renderRegionalPartnerSelect(validation) {
+    if (this.shouldRenderRegionalPartnerSelect()) {
+      const options = window.dashboard.workshop.available_regional_partners.map((partner) => {
+        return (<option key={partner.id} value={partner.id}>{partner.name}</option>);
+      });
+      // we always present an empty placeholder for an optional field
+      const placeHolder = <option />;
+      return (
+        <FormGroup validationState={validation.style.regional_partner_id}>
+          <ControlLabel>Regional Partner</ControlLabel>
+          <FormControl
```

Comment:
Names are untruncated in the dropdown:

![image](https://cloud.githubusercontent.com/assets/244100/23774180/5133d66c-04d7-11e7-8d97-beffda7a00df.png)

But truncated when selected:

![image](https://cloud.githubusercontent.com/assets/244100/23774165/46146be8-04d7-11e7-88fb-0dfa7fd4a1a9.png)